src/quo2/components/selectors/react_selector_add/view.cljs

```diff
@@ -0,0 +1,44 @@
+(ns quo2.components.selectors.react-selector-add.view
+  (:require [quo2.components.icon :as icons]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as theme]
+            [react-native.core :as rn]))
+
+(defn- get-color
+  [pinned-colors default-colors pinned? theme]
+  (apply colors/theme-colors
```

Comment:
perhaps just me, but this code seems a bit complicated and might be more readable if kept simple

Reply:
It was a bit uglier otherwise, but makes sense. Would rather have it more readable since it'll only be used in these two places.